README.md

```diff
@@ -189,6 +190,28 @@ Content-Type: application/json
 | purpose     | Purpose of document signature                                                                   | Mandate as per norms. Will be used to generate consent text and logged in DB. |
 | responseURL | POST API URL where the Agency receives the response after the e-signing is completed.           | A valid POST API URL, else response back to your server will fail.            |
 
+<a name="esignSignAllPages"></a>
+
+#### 4.3.1 Sign on All Pages
+
+You can also attach signature on all the pages of the document. The signature will be attached on the **same coordinate of every page**. In order to attach the signature on every page you need to provide `signPageNumber` as 0 in the init call.
+
+<pre><code class="language-json">{
```

Comment:
@iAziz786 Add co-ordinate part in the example so it gets following instead of drag to place signature option.